### ISSUE TITLE
refreshing page always goes to /login

### ISSUE THREAD

**Issue by chovy** [ItalyPaleAle/svelte-spa-router#38]
Clicking around works as expected but if i'm on say /pricing page and refresh i get the /login page for some reason.

```
import Home from './components/Home.svelte';
import Login from './components/Login.svelte';
import Signup from './components/Signup.svelte';
import Pricing from './components/Pricing.svelte';
import Forgot from './components/Forgot.svelte';
import Connections from './components/Connections.svelte';

const routes = {
    '/': Home,
    '/pricing': Pricing,
    '/login': Login,
    '/signup': Signup,
    '/forgot': Forgot,
    '/connections': Connections,
};

export default routes;
```

**Comment by hmmhmmhm** [comment]
there are no chrome inspector errors?

**Comment by ItalyPaleAle** [comment]
@chovy Can you please create a repo (or a Svelte REPL) to reproduce this?

**Comment by TorstenDittmann** [comment]
@chovy if you are willing to share your repo where this login redirect stuff happens, I am willing to take a deeper look into it tomorrow.

**Comment by chovy** [comment]
sure, send me your email: anthony@profullstack.com

**Comment by TorstenDittmann** [comment]
Looked into this. This bug is not related to svelte-spa-rotuer but can be solved when a solution for #23 arrives :)
